pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1179,11 +1179,49 @@ func TestSchedulerCache_SnapshotNodeInfo(t *testing.T) {
 		updatedPods = append(updatedPods, updatedPod)
 	}
 
-	type operation struct {
-		operator string
-		index    int
+	var cache *schedulerCache
+	var snapshot NodeInfoSnapshot
+	addNode := func(i int) func() {
```

Comment:
Oh, optionally you could write this as `addNode := func(i int) operation {`

Reply:
Done. Thanks for your review!